pkg/apis/nsx.vmware.com/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	AccessModePublic    string = "Public"
-	AccessModePrivate   string = "Private"
```

Comment:
No, we should change "Project" to "PrivateTGW", not this one.

Also let us follow Golang/K8s convention to use PrivateTGW, not Private_TGW.

Reply:
I just saw the new comments and the commit was updated yesterday, will update again later.